wireguard-vpn/js/view/Settings.js

```diff
@@ -72,14 +72,24 @@ Ext.define('Ung.apps.wireguard-vpn.view.Settings', {
             hidden: true,
             margin: '18 0 5 5'
         }],
+    }, {
+        xtype: "checkbox",
+        bind: '{settings.mapTunnelDescUser}',
+        fieldLabel: 'Map tunnel description to users'.t(),
```

Comment:
Change to "Authenticate connected username as tunnel description"

Reply:
Changed verbiages as per discussion
Label is changed to `Set authenticated username as tunnel description`
qtip message changed to `If enabled, tunnel description will be set as authenticated user`